src/ext/BalExtension/wixstdba/WixStandardBootstrapperApplication.cpp

```diff
@@ -991,6 +992,26 @@ class CWixStandardBootstrapperApplication : public CBalBaseBootstrapperApplicati
         return __super::OnExecuteFilesInUse(wzPackageId, cFiles, rgwzFiles);
     }
 
+protected: // internals
+    HRESULT FindLocFile(
+        __in_z LPCWSTR wzBasePath,
+        __in_z LPCWSTR wzLocFileName,
+        __in_z_opt LPCWSTR wzLanguage,
+        __inout LPWSTR* psczPath
+        )
+    {
+        LONGLONG llLangFix = 0;
+        BalGetNumericVariable(WIXSTDBA_VARIABLE_LANGUAGE_FIX, &llLangFix);
+        if (llLangFix)
+        {
+            return LocProbeForFileFixed(wzBasePath, wzLocFileName, wzLanguage,psczPath);
+        }
+        else
+        {
+            return LocProbeForFile(wzBasePath, wzLocFileName, wzLanguage,psczPath);
```

Comment:
whitespace matches the rest of the entire file, except for the double blank lines between methods. In my next push, that will be resolved.